### DISCUSSION TITLE
Is it not possible to load a JSON preset in Davinci Resolve?

### DISCUSSION THREAD

**Discussion by joman66** [ntsc-rs/ntsc-rs#211] [Q&A]
Asking as I only know how to load "settings" and not the JSON presets that people graciously post here in the discussions. Thank you.

**Comment by TheSystemGuy1337** [comment]
This is an independent program. Unless there is a variant of NTSC-RS that is a plugin for Davinci Resolve, you simply cannot do that.

**Reply by valadaptive** [reply]
[There is.](https://ntsc.rs/docs/openfx-plugin/)

**Reply by TheSystemGuy1337** [reply]
I didn't even know that.

**Comment by valadaptive** [comment]
Tracking this feature in https://github.com/valadaptive/ntsc-rs/issues/159.
